test/include/dlaf_test/matrix/matrix_local.h

```diff
@@ -97,6 +97,14 @@ struct MatrixLocal<const T> {
     return GlobalElementSize{layout_.size().rows(), layout_.size().cols()};
   }
 
+  TileElementSize tile_size() const noexcept {
+    return layout_.blockSize();
```

Comment:
The layout member functions will presumably be renamed separately, right?

Reply:
Yes, `Layout` will get its refactoring as well.